### ISSUE TITLE
Pass allowDuplicate Option?

### ISSUE THREAD

**Issue by LazerJesus** [anki-editor/anki-editor#26]
The `addNote` endpoint allows to pass an option called `allowDuplicate`, which i would like to enable. 
Is there a way to pass options? 

[note actions docs for ref](https://foosoft.net/projects/anki-connect/#note-actions)

**Comment by FinnFrotscher** [comment]
It would be here If not mistaken: 
```
    (make-anki-editor-note :id note-id
                           :model note-type
                           :deck deck
                           :tags tags
                           :fields exported-fields)))
```

**Comment by orgtre** [comment]
Thanks, this would be a useful option to have. 

I think it might be enough to edit `anki-editor-api--note` to add an options entry containing allowDuplicate, like this I think: `:options (list :allowDuplicate anki-editor-allow-duplicate)` where anki-editor-allow-duplicate is a customize variable that can be set to 'true or 'false. Compare with the sample request in the docs you link.

Feel free to send a pull request, otherwise I’ll do it once I find some time.

**Comment by slotThe** [comment]
Closed by #28
